template.yml

```diff
@@ -19,10 +19,6 @@ Parameters:
     Type: String
     Default: ""
     Description: DO NOT CHANGE unless you know what you are doing. Override the default location of the function source code.
-  PermissionsBoundaryArn:
```

Comment:
I don't think it hurt to keep this parameter? Some customers does need this to ensure the IAM roles created have permissions scoped by a boundary.